### PR TITLE
adds a branding page

### PR DESCRIPTION
**Pull request by ln1draw**
Adds branding page (and associated routes, styling, etc), and the method pretty_reference

### CODE REVIEW

File: app/assets/stylesheets/static.scss [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+
+.branding {
+  p {
+    min-height: 24px;
+    margin: 24px;
+
+    .left {
+      float: left;
+      font-weight: bold;
+    }
```
Comment (krainboltgreene) [inline app/assets/stylesheets/static.scss]: In the future we should replace this with grid or flex. It'll be less effort to handle different window sizes.

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -8,4 +8,7 @@
   <a href="/notechforice">
     <h2>Why No Tech For ICE</h2>
   </a>
+  <a href="/branding">
+    <h2>Branding Info</h2>
+  </a>
```
Comment (krainboltgreene) [inline app/views/layouts/_header.html.erb]: Should we use the built in url helpers for these or is the goal to make it easier to transition to a frontend client?

File: app/views/locations/index.html.erb [hunk 3]
```diff
@@ -9,7 +9,7 @@
   <ul>
     <% @locations.each do |location| %>
       <li class="location">
-        <%= link_to(location.name + ' - ' + location.pretty_address, locations_show_path(id: location.id)) %>
+        <%= link_to(location.pretty_reference, locations_show_path(id: location.id)) %>
```
Comment (krainboltgreene) [inline app/views/locations/index.html.erb]: Great change!!

Comment (krainboltgreene) [reply app/views/locations/index.html.erb]: FYI, you can do `location_path(location)`.

File: config/routes.rb [hunk 4]
```diff
@@ -2,5 +2,6 @@
   get '/', to: 'locations#index'
   get 'locations/show'
   get '/notechforice', to: 'static#notech'
+  get '/branding', to: 'static#branding'
```
Comment (krainboltgreene) [inline config/routes.rb]: It might be worthwhile to use the high-voltage gem.

File: app/assets/stylesheets/static.scss (hunk 1 above)
Comment (ln1draw) [reply app/assets/stylesheets/static.scss]: That's a good comment for the github issue on CSS implements!

File: app/views/layouts/_header.html.erb (hunk 2 above)
Comment (ln1draw) [reply app/views/layouts/_header.html.erb]: There wasn't a specific goal one way or the other, I think I just used this bc I forget the fiddly bits in the erb helpers sometimes/often. Do we need these static links to be helpers when it's just static data?

(this is a sincere question--I actually don't know why you would choose one over the other; this just seems like less processing power and also less human work)

File: config/routes.rb (hunk 4 above)
Comment (ln1draw) [reply config/routes.rb]: I don't know what that is or why I would use it? More words on that comment if you want to make that suggestion, please.

**Comment by ln1draw** [comment]
Thank you for your work and your time @krainboltgreene !! much appreciated :D :D :D

**Pull request merged**